updating/updating-clusters-sro.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-sro"]
+= Updating a cluster that includes the SRO
```

Comment:
same comment as above: updating to upgrading

Reply:
guideline
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update says "update" is the preferred term.